contracts/modules/assessment/IndividualClaims.sol

```diff
@@ -230,12 +230,31 @@ contract IndividualClaims is IIndividualClaims, MasterAwareV2 {
     uint16 segmentId,
     uint96 requestedAmount,
     string calldata ipfsMetadata
-  ) external payable override onlyMember whenNotPaused returns (Claim memory) {
-    require(
+  ) external payable override onlyMember whenNotPaused returns (Claim memory claim) {
+    claim = _submitClaim(coverId, segmentId, requestedAmount, ipfsMetadata, msg.sender);
+    return claim;
+  }
+  function submitClaimOf(
+    uint32 coverId,
+    uint16 segmentId,
+    uint96 requestedAmount,
+    string calldata ipfsMetadata,
+    address owner
+  ) external payable override onlyInternal whenNotPaused {
+    _submitClaim(coverId, segmentId, requestedAmount, ipfsMetadata, owner);
+  }
+
+  function _submitClaim(
+    uint32 coverId,
+    uint16 segmentId,
+    uint96 requestedAmount,
+    string calldata ipfsMetadata,
+    address owner
+  ) internal whenNotPaused returns (Claim memory) {
```

Comment:
no need for whenNotPaused here, it's an internal function, the care is for the publicly exposed functions